### PR TITLE
Feature/repo config

### LINKED ISSUE #208: [SupportPlatform-Web] Implement a config interface integration on button to enable/disable repository auto sync (image on description of this issue)
Image

### LINKED ISSUE #209: [SupportPlatform-Service] Implement PATCH to enable/disable branch auto sync

### LINKED ISSUE #210: [Supportplatform-Web] Implement a config interface to enable or disable branches to be synchronize in repository screen

### LINKED ISSUE #212: [Supportplatform-Web] Implement the date of last sync on repository interface

### LINKED ISSUE #215: [Supportplatform-Web] Implement visualization of which Evaluation Method is associated in the Repository

### PR DESCRIPTION
**Pull request by guizombas**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

closes #208 
closes #209 
closes #210 
closes #215 
closes #212

### CODE REVIEW

File: Codigo/supportplatform-service/src/service/BranchService.ts [hunk 1]
```diff
@@ -28,7 +30,13 @@ export default class BranchService {
 
             logger.info("Searching for all branches");
             const { rows, count } = await Branch.findAndCountAll({
```
Comment (Lucas-Angelo) [inline Codigo/supportplatform-service/src/service/BranchService.ts:32]: Adicionar orderBy commitAutomaticSynchronization e fileAutomaticSynchronization

File: Codigo/supportplatform-service/src/test/controller/BranchController.test.ts [hunk 2]
```diff
@@ -158,3 +158,247 @@ describe("GET repository/{repositoryId}/branch", () => {
         expect(response.body.message).toBe("Invalid token");
     });
 });
+
+describe("PATCH /repository/{id}/branch/{branchId}", () => {
```
Comment (Lucas-Angelo) [inline Codigo/supportplatform-service/src/test/controller/BranchController.test.ts:162]: Só 1 dúvida, para PATCH de automatic_sync do /repository já tem teste?

**Review by Lucas-Angelo — CHANGES_REQUESTED** [review]
- Adicionar em todas as opções: “você realmente deseja fazer isso” na tela de config de repositório

File: Codigo/supportplatform-service/src/test/controller/BranchController.test.ts (hunk 2 above)
Comment (guizombas) [reply Codigo/supportplatform-service/src/test/controller/BranchController.test.ts:162]: tem, ce já tinha feito
image

**Review by Lucas-Angelo — APPROVED** [review]

**Pull request merged**